src/keeper/contracts/conditions/Condition.abstract.ts

```diff
@@ -20,6 +20,7 @@ export interface ConditionContext {
   service: Service
   price: AssetPrice
   creator: string
+  serviceReference?: number
```

Comment:
what is the `serviceReference`? and how are they different from `serviceIndex` and the `index` key of a service?

Reply:
removed